### PR TITLE
Homeui

### PR DESCRIPTION
**Pull request by rahulraut1588**

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -1,4 +1,9 @@
-body {
-  padding-right: 20px;
-  padding-left: 20px;
-}
+.container-fluid.back-color {background: radial-gradient(#e8e8e8, #2e6da4);padding: 240px 0px;}
+body { display: table; width: 100%; height: 100%; }
```
Comment (Shrikant9) [inline src/css/style.css]: linting

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -7,10 +7,53 @@ export default class Home extends React.Component {
...
     return (
-      <div className="container">
-        <h1>Hello World!</h1>
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-        <Link to='/about'> About Page </Link>
+
+      <div className="container-fluid back-color">
+
+          <div className="container">
+
```
Comment (Shrikant9) [inline src/pages/Home.jsx]: no new lines needed.

File: src/css/style.css [hunk 3]
```diff
@@ -1,4 +1,44 @@
 body {
-  padding-right: 20px;
-  padding-left: 20px;
+  display: table;
+  width: 100%;
+  height: 100%;
+}
```
Comment (Shrikant9) [inline src/css/style.css]: New line after block.

Comment (rahulraut1588) [inline src/css/style.css]: on it

File: src/css/style.css [hunk 4]
```diff
@@ -1,4 +1,51 @@
...
+  background: #f0ad4e;
+  margin: 30px auto;
+  padding: 10px 35px;
+  border: 1px solid #444;
+}
+
+.dummy-content {
+  margin-top: 40px;
+  padding-left: 15px;
 }
```
Comment (Shrikant9) [inline src/css/style.css]: New Line before EOF.

Comment (rahulraut1588) [inline src/css/style.css]: New lines are already added but not visible on split screen.

**Pull request merged**